artifact-app/build.rs

```diff
@@ -53,8 +55,21 @@ fn check_deps() {
 
 fn build_mdbook() {
     println!("Building the book");
+
+    let mdbook_dir = if cfg!(windows) {
+
+        let regex = Regex::new("[a-zA-Z].+").unwrap();
+        let dir_as_str = BOOK.as_path().to_str().expect("Invalid book dir");
+        let shortened = regex.find(dir_as_str).unwrap();
```

Comment:
just return `regex.find(dir_as_str).unwrap().as_str()` instead of taking the slice.

Reply:
https://docs.rs/regex/1.1.9/regex/struct.Match.html#method.as_str

Reply:
oh, that's a really neat feature.